lib/routes/charge/items/item_page.dart

```diff
@@ -246,29 +261,48 @@ class CreateItemPageState extends State<CreateItemPage> {
         backgroundColor: Theme.of(context).canvasColor,
         leading: backBtn.BackButton(),
         title: Text(
-          "Add Item",
+          "$_titlePrefix Item",
           style: Theme.of(context).appBarTheme.textTheme.title,
         ),
         actions: actions == null ? <Widget>[] : actions,
         elevation: 0.0,
       ),
       body: body,
       bottomNavigationBar: SingleButtonBottomBar(
-        text: "Add Item",
+        text: "$_titlePrefix Item",
         onPressed: () {
           {
             if (_formKey.currentState.validate()) {
-              AddItem addItem = AddItem(
-                Item(
-                  name: _nameController.text.trimRight(),
-                  currency: _currencySymbol(),
-                  price: double.parse(_formattedPrice()),
-                ),
-              );
-              widget._posCatalogBloc.actionsSink.add(addItem);
-              addItem.future.then((_) {
-                Navigator.pop(context);
-              });
+              if (_inEditMode) {
+                UpdateItem updateItem = UpdateItem(
+                  Item(
```

Comment:
I think it is cleaner to merge changes with the current edited item.
Something like:
`widget.item.copyWith(name: ..., price: ..., )`

Reply:
[Done](https://github.com/breez/breezmobile/pull/296/commits/ad6d6e0ee983b18b1315650feaa6802af10babac)